tests/cross_functional/performance/csi_tests/test_bulk_pod_attachtime_performance.py

```diff
@@ -232,6 +233,15 @@ def test_bulk_pod_attach_performance(self, interface_type, bulk_size):
             "test_time", {"start": test_start_time, "end": test_end_time}
         )
 
+        start_t = time.strptime(test_start_time, "%Y-%m-%dT%H:%M:%SGMT")
+        epoch_gmts = calendar.timegm(start_t)
+        end_t = time.strptime(test_end_time, "%Y-%m-%dT%H:%M:%SGMT")
+        epoch_gmte = calendar.timegm(end_t)
+
+        self.test_duration = int(epoch_gmte - epoch_gmts)
+
+        self.deploy_odf_grafana()
```

Comment:
If grafana is already deployed do we still attempt to re-deploy?

Reply:
I am checking for grafana pods status, if we have pods running it will continue with the rest of the operation. Installation will be one time activity for the entire run